Changelog.md

```diff
@@ -32,6 +32,7 @@ __New Features__
 - Revises shared mechanical ventilation preconditioning control logic to operate less often.
 - Removes error-check for number of bedrooms based on conditioned floor area, per RESNET guidance.
 - Updates the reporting measure to register all outputs from the annual CSV with the OS runner (for use in, e.g., PAT).
+- Use constant Cutler curve coefficients in installation quality calculations instead of coil curve coefficients for two-speed, variable-speed systems and ground source heat pump.
```

Comment:
Is there anything else from this PR that should go in the changelog? The PR description says "To fix an issue where quadlinear/quintlinear curve is not reaching 1 for rated airflow in EMS program."

Reply:
That language is from using ground source heat pump curve coefficients in IQ calculation, I think it's no longer applicable, I should have removed it from description.